pages/linux/readpe.md

```diff
@@ -0,0 +1,28 @@
+# readpe
+
+> Displays information about PE files.
+> More information: <https://manpages.debian.org/testing/pev/readpe.1.en.html>.
```

Comment:
```suggestion
> More information: <https://manpages.debian.org/latest/pev/readpe.1.html>.
```

Reply:
Why not?
```suggestion
> More information: <https://manned.org/readpe>.
```

Reply:
Right, forgot about that. And it doesn't seem like `readpe` comes from debian.